data/hai/unfettered 0 prologue.txt

```diff
@@ -516,57 +521,1009 @@ mission "Unfettered Jump Drive Trading"
 
 
 
-mission "Unfettered returning home"
-	description "This Hai has asked you to smuggle him out of Unfettered space and bring him to <destination>."
-	minor
-	source
-		attributes "unfettered"
-	destination "Hai-home"
-	clearance
-	passengers 1
+mission "<uhai> 0: Bring Hai"
+	name `Bring True Hai`
+	description `Bring <bunks> Hai from <stopovers> to <origin>.`
+	source "Firelode"
+	stopover "Snowfeather"
+	passengers 10
+	blocked "You will need room for <bunks> <passengers> in order to take the next Unfettered mission."
 	to offer
-		has "Unfettered Tribute 3: done"
-		random < 40
+		"reputation: Hai (Unfettered)" >= 0
+		# guaranteed to offer the first time and then 20% chance
+		or
+			not "unfettered: offered bring hai"
+			random < 20
+	on defer
+		set "unfettered: offered bring hai"
+	substitutions
+		# Translate it if you have a translator, otherwise someone will offer to do it for you.
+		<firstgroup> "but because it is in Hai you cannot make sense of the words. However even from afar you can see how malnurished the children are."
```

Comment:
```suggestion
		<firstgroup> "but because it is in Hai you cannot make sense of the words. However even from afar you can see how malnourished the children are."
```